src/components/Modals/NewCourse/SelectedRequirementEditor.vue

```diff
@@ -1,33 +1,44 @@
 <template>
   <div>
     <div v-if="requirementsThatAllowDoubleCounting.length > 0">
-      <div class="newCourse-title">
-        This class automatically fulfills the following requirement(s):
-      </div>
+      <div class="newCourse-title">This class fulfills the following requirement(s):</div>
       <div class="newCourse-requirements-container">
         <div class="newCourse-requirements">
-          {{ requirementsThatAllowDoubleCounting.join(', ') }}
+          {{ editMode ? requirementsThatAllowDoubleCounting.join(', ') : chosenRequirementText }}
```

Comment:
> Nothing is autofilled, but then when you press "Select Requirements", liberal studies is

I believe this line (more specifically `chosenRequirementText()`) points to the issue that @einc brought up.  `chosenRequirementText` is `''` if `this.selectedRequirementID === ''`.

In the video, you see that there is no text shown under "This class fulfills the following requirement(s):" even though when `editMode` is `true`, we see that CS 2110 fulfills Engineering Distribution and Introductory Programming. Only when we select a requirement (i.e. `this.selectedRequirementID !== ''`) do we actually see Engineering Distribution and Introductory Programming appear (in addition to the requirement we selected). 

https://user-images.githubusercontent.com/54298311/113088159-0f7dac80-91b3-11eb-9da6-74ead897f030.mov


This second video shows that this bug is more of how text is displayed since the course still ends up applying to Engineering Distribution and Introductory Programming even though it doesn't appear to do so based on the text in the add modal.

https://user-images.githubusercontent.com/54298311/113088170-13113380-91b3-11eb-8638-7c91be5a2bd0.mov